.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's not commit anything in `.idea` directory.
You can add `.idea` in `vim .git/info/exclude`.